django/tasks/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .base import Task, get_backend  # NOQA
+from .backends import TaskResult  # NOQA
+from .constants import *  # NOQA
+from .register import registry
+
+
+def task(name_or_func=None, name=None, using=None, options=None):
+    # @task
+    if callable(name_or_func):
+        if isinstance(name_or_func, Task):
```

Comment:
What about just checking if it has a callable `run` attribute instead? Do we have to enforce that it be a subclass of `Task`?